.github/workflows/algolia-sync-v2.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
       - name: Build
-        run: yarn exec "yarn workspaces foreach -vptR --from '{@asap-hub/asap-cli,@asap-hub/asap-server}' run build:babel"
+        run: yarn exec "yarn workspaces foreach -vptR --from '{@asap-hub/asap-cli,@asap-hub/crn-server}' run build:babel"
```

Comment:
What about renaming asap-cli? Is it specific to crn?

Reply:
its not